pyshop/shopcore/shop_rest.py

```diff
@@ -0,0 +1,35 @@
+import requests
+import json
+import numpy
+import pandas
```

Comment:
numpy og pandas importeres med alias 'np' og 'pd' andre steder i koden, så du kan jo gjøre det her også for konsistens :)